src/posix/net.cpp

```diff
@@ -718,14 +720,19 @@ CurlHttpIO::~CurlHttpIO()
     closecurlevents(PUT);
 
     curlMutex.lock();
```

Comment:
We shouldn't need this anymore

Reply:
Actually, we do to ensure that nothing else runs while we shutdown. But then why do we need the int to be atomic?

Reply:
I made the int atomic since threading is relevant for it - probably it's not strictly needed since its use is inside the mutex, but it won't hurt and it's only used on startup/shutdown.  Perhaps we could get away without locking the mutex on exit (in which case we do need the atomic), but I think it's nicer for the startup and shutdown to be symmetric.  I think you'd agree it will work?  thanks

Reply:
Yes it works but it's confusing to the reader that you lock a mutex and also use an atomic for no apparent reason.

Reply:
ok, atomic<> removed